packages/block-library/src/post-comments-link/index.php

```diff
@@ -48,7 +48,7 @@ function render_block_core_post_comments_link( $attributes, $content, $block ) {
 		);
 	}
 
-	return "<div {$wrapper_attributes}><a href='{$comments_link}'>{$comment_text}</a></div>";
+	return '<div ' . $wrapper_attributes . '><a href=' . esc_url( $comments_link ) . '>' . esc_html( $comment_text ) . '</a></div>';
```

Comment:
Escaping the `$comment_text` here would probably cause an a11y issue since it contains a `<span class="screen-reader-text">` element.

Reply:
Yeh I think we need more finesse here. Perhaps `wp_kses_post()`?

Reply:
I've moved the escaping to the point at which the variable is built and also renamed it to be clearer that it isn't just text.